src/opcodes/tests/test_stack.cairo

```diff
@@ -269,3 +269,12 @@ fn test_op_2rot_insufficient_items() {
     let mut engine = utils::test_compile_and_run_err(program, Error::STACK_OUT_OF_RANGE);
     utils::check_dstack_size(ref engine, 5);
 }
+
+#[test]
+fn test_opcode_sha256() {
+    let program = "OP_1 OP_SHA256";
+    let mut engine = utils::test_compile_and_run(program);
+    utils::check_dstack_size(ref engine, 1);
+    let expected_dstack = array!["1274352175"];
```

Comment:
I believe the resulting stack should be `array![hex_to_bytecode("0x4bf5122f344554c53bde2ebb8cd2b7e3d1600ad631c385a5d7cce23c7785459a")]`.
If you use this ide https://ide.scriptwiz.app, you can change the witness script to `OP_1 OP_SHA256` to see and get other examples too.

Reply:
I see... working on it